File: 3. Semestar/Napredno programiranje i programski jezici/Vezbe/Vezbe 3/1/krug.cpp
#include "krug.hpp"

Krug::Krug() {
    r = 1;
}

Krug::Krug(double r = 1) {
    this->r = r;
}

double Krug::getR() {
    return r;
}

double Krug::getO() {
    return 2 * r * 3.14;
}

double Krug::getP() {
    return r * r * 3.14;
}

File: 3. Semestar/Napredno programiranje i programski jezici/Vezbe - Nasledjeno/vezbe9/Njihova resenja/3/Krug.java
package z3;

public class Krug {
	private double r;
	
	public Krug(double r) {
		this.r = r;
	}
	
	public Krug(Krug k) {
		this.r = k.r;
	}

	public double getR() {
		return r;
	}

	public void setR(double r) {
		this.r = r;
	}	
	
	public double getO() {
		return 2 * Math.PI * r;
	}
	
	public double getP() {
		return Math.pow(r, 2) * Math.PI;
	}

}
